plugin/al/MIGRATION_GUIDE.md

```diff
@@ -0,0 +1,348 @@
+
+## Commands
+
+### Import Command and Export Command
+
+#### `AL_usdmaya_ImportCommand`
+
+ADSK_USDMaya provided an equivalent command `mayaUSDImportCommand`, see [Import and Export](#import-and-export-static-io) below for more details.
+
+#### `AL_usdmaya_ExportCommand`
+
+ADSK_USDMaya provided an equivalent command `mayaUSDExportCommand`, see [Import and Export](#import-and-export-static-io) below for more details.
+
+### Proxy Shape commands
+
+#### `AL_usdmaya_ProxyShapeImport`
+
+This command does not only create the proxy shape node but also loads the given USD stage; in ADSK_USDMaya, you will have to do it separately:
+
+- Create the proxy shape node
+- Set the `.filePath` attribute to point to your USD root layer, and set other attributes if necessary
+- Connect `time1` to `.time` attribute if you need to read animation
+
+Example MEL function `mayaUsd_createStageFromFilePath` bundled in ADSK_USDMaya can be found [here](https://github.com/Autodesk/maya-usd/blob/release/v0.27.0/plugin/adsk/scripts/mayaUsd_createStageFromFile.mel#L274C20-L274C51).
+
+#### `AL_usdmaya_ProxyShapeResync`
+
+[`AL_usdmaya_ProxyShapeResync`](https://github.com/Autodesk/maya-usd/blob/release/v0.27.0/plugin/al/lib/AL_USDMaya/AL/usdmaya/cmds/ProxyShapeCommands.h#L198) triggers AL's [`ProxyShape::resync()`](https://github.com/Autodesk/maya-usd/blob/release/v0.27.0/plugin/al/lib/AL_USDMaya/AL/usdmaya/nodes/ProxyShape.cpp#L806) which will then rerun the [translator system](#translator-plugin-system) to import/teardown/update associated Maya data according to the changes in USD stage, you will need to explicitly traverse stage hierarchy manually to achieve the same in ADSK_USDMaya (see [translator system](#translator-plugin-system) above for the details).
+
+#### `AL_usdmaya_TranslatePrim`
+
+ADSK_USDMaya introduced a new ["Edit As Maya"](https://help.autodesk.com/view/MAYAUL/2024/ENU/?guid=GUID-C1C08BA5-24EA-44FF-9497-71E0A6843744) workflow to exchange data between USD and Maya, there are two equivalent Maya commands to push / pull USD data to/from Maya:
+
+- `mayaUsdEditAsMaya`
+  - Equivalent implementation of AL's `AL_usdmaya_TranslatPrim` where "importPaths" or "updatePaths" or "forceImport" flag is being used, a.k.a "Push to Maya"
+
+- `mayaUsdMergeToUsd`
+  - Equivalent implementation of AL's `AL_usdmaya_TranslatPrim` where "teardownPaths" flag is being used, a.k.a "Pull to USD"
+
+For more info about this new workflow in Maya, see the official ["Edit As Maya"](https://help.autodesk.com/view/MAYAUL/2024/ENU/?guid=GUID-C1C08BA5-24EA-44FF-9497-71E0A6843744) workflow doc.
```

Comment:
Because Maya 2025 has already released should we give the 2025 link?
https://help.autodesk.com/view/MAYAUL/2025/ENU/?guid=GUID-C1C08BA5-24EA-44FF-9497-71E0A6843744

Reply:
All the links have been updated to 2025 (https://github.com/Autodesk/maya-usd/pull/3738/commits/f17d8eea9e48afbea9e3b971badc2d6f4a084381)